testsuite/lb_get_u_at_part.py

```diff
@@ -0,0 +1,78 @@
+import sys
+import unittest as ut
+import numpy as np
+import numpy.testing
+import espressomd
+from espressomd import lb
+
+
+@ut.skipIf(not espressomd.has_features(["LB_GPU"]),
+           "LB_GPU feature not available, skipping test!")
+class TestLBGetUAtPart(ut.TestCase):
+    """
+    Check velocities at particle positions are sorted by ``id`` and
+    quantitatively correct (only LB GPU).
+
+    """
+    @classmethod
+    def setUpClass(self):
+        self.params = {
+            'tau': 0.01,
+            'agrid': 0.5,
+            'box_l': [12.0, 12.0, 12.0],
+            'dens': 0.85,
+            'viscosity': 30.0,
+            'friction': 2.0,
+            'gamma': 1.5
+        }
+        self.system = espressomd.System()
+        self.system.box_l = self.params['box_l']
+        self.system.cell_system.skin = 0.4
+        self.system.time_step = 0.01
+        self.n_nodes_per_dim = int(self.system.box_l[0] / self.params['agrid'])
+        for p in range(self.n_nodes_per_dim):
+            if p % 2 == 0:
+                self.system.part.add(id=p,
+                                     pos=[0.5 * self.params['agrid'] * (p + 1),
+                                          0.5 * self.params['agrid'],
+                                          0.5 * self.params['agrid']])
+        self.lb_fluid = lb.LBFluid_GPU(
+            visc=self.params['viscosity'],
+            dens=self.params['dens'],
+            agrid=self.params['agrid'],
+            tau=self.params['tau'],
+            fric=self.params['friction']
+        )
+        self.system.actors.add(self.lb_fluid)
+        self.vels = np.zeros((self.n_nodes_per_dim, 3))
+        self.vels[:, 0] = np.arange(self.n_nodes_per_dim, dtype=float)
```

Comment:
If I use
```python
self.vels[:, 0] = np.ones(self.n_nodes_per_dim, dtype=float)
```
then the result with 3pt coupling is off by a constant factor of exactly `2.25`.  I have still no idea why.